src/components/home.js

```diff
@@ -1107,7 +1107,7 @@ class home extends Component {
             <span> Gopa Vasanth </span>
           </a>
           and <b>Hassan Amin</b> &hearts; |
-          <a href="https://github.com/gopavasanth/video-cut-tool-front-end">
+         <a href="https://github.com/gopavasanth/video-cut-tool-front-end">
```

Comment:
You've not changed this.

Reply:
I tried, but if I change this, error 404 would come out

Reply:
That's why I didn't change current links, I tried changing all of them but error 404 would come out after deleting "-front-end"

Reply:
Oh, you're right. This will involve changing the repo name, and only owner can do that. (I noted so in the [Phabricator task](https://phabricator.wikimedia.org/T239132#5702563). Thank you.

Reply:
>_< No thanks